test/e2e/flask/solana/create-solana-account.spec.ts

```diff
@@ -0,0 +1,72 @@
+import { Suite } from 'mocha';
+import HeaderNavbar from '../../page-objects/pages/header-navbar';
+import AccountListPage from '../../page-objects/pages/account-list-page';
+import { ACCOUNT_TYPE } from '../../page-objects/common';
+import { withSolanaAccountSnap } from './common-solana';
+
+describe('Create/Remove Solana Account', function (this: Suite) {
+  it('Creates 2 Solana accounts', async function () {
+    await withSolanaAccountSnap(
+      { title: this.test?.fullTitle() },
+      async (driver) => {
+        // check that we have one Solana account
+        const headerNavbar = new HeaderNavbar(driver);
+        await headerNavbar.check_pageIsLoaded();
+        console.log('Page is loaded');
```

Comment:
we don't need the console log here, because we already have it in `check_pageIsLoaded`, otherwise we will have duplicate condole log. see here: https://github.com/MetaMask/metamask-extension/blob/37e79458f3fb8299be2c5d583796fd79f28cb60b/test/e2e/page-objects/pages/header-navbar.ts#L39
We always prioritize to add console log in class methods rather than in test spec, because it will be reusable whenever the methods are called by test spec.

Reply:
fixed